modules/ROOT/pages/automated-policies-landing-page.adoc

```diff
@@ -0,0 +1,22 @@
+= Automated Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Automated policies make it easy for your API management team to satisfy compliance in common security, and logging requirements by automatically applying the same set of provided policies to all of your configured deployed runtimes. +
+Your API development team can focus on building API proxies without worrying about compliance requirements because all of your deployed API proxies would inherit the configured automated policy for their runtime versions.
+
+Automated policies ensure consistency and reduce possible errors because an administrator no longer needs to remember to manually insert a policy to enforce common standards.
+
+All of the xref:policies-ootb-landing-page.adoc[Mule 4 Provided Policies] can be configured as automated policies. +
+Policies can be configured to be applied at all the API’s running under a business group/environment.
```

Comment:
For now, an automated policy will be applied to a certain environment (not for the entire business group)

Reply:
Changed that: 853ecd2